src/Utility/Text.php

```diff
@@ -891,7 +898,9 @@ public static function excerpt(string $text, string $phrase, int $radius = 100,
      */
     public static function toList(array $list, ?string $and = null, string $separator = ', '): string
     {
-        $and ??= __d('cake', 'and');
+        static::$_useI18n ??= function_exists('__d') === true;
+        $and ??= (static::$_useI18n === true ? __d('cake', 'and') : 'and');
```

Comment:
```suggestion
        static::$useI18n ??= function_exists('__d');
        $and ??= static::$useI18n ? __d('cake', 'and') : 'and');
```

Comparing booleans with a boolean is redundant, directly use the boolean value.

Reply:
As you want, the comparison is safer and for more clarity in reading the code.